### DISCUSSION TITLE
Support system Trash/Recycle Bin

### DISCUSSION THREAD

**Discussion by mon-jai** [spacedriveapp/spacedrive#2296] [Feature Request]
![image](https://github.com/spacedriveapp/spacedrive/assets/91261297/c92c95c8-b673-40ed-a881-c8729ca7f2eb)

It shouldn't be hard to implement. When someone "delete" a file, just move it to `C:/$Recycle` (or macOS / Linux equivalent) instead.

**Comment by utkubakir** [comment]
Thank you for the suggestion, our delete file flow will get better soon.
